ports/libexif/portfile.cmake

```
@@ -1,14 +1,12 @@
-include(vcpkg_common_functions)
-
 if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
     message(FATAL_ERROR "libexif currently only supports being built for desktop")
 endif()
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "uwp")
```
We prefer to use this function to post the message. Could you please update this?

---

Done. What about the `Supports` field in the `CONTROL` file? Does that relate to this? I guess I shouldn't use that if you are phasing out the `CONTROL` files and moving to the manifests at some point?

---

This is related with `Supports `field in the `CONTROL` file. We might drop out `CONTROL` file in the near future but not for now.
So you don't need to change anything about this.